### PR TITLE
feat(.github/actions): add `get-new-env-vars-v1` action

### PR DESCRIPTION
**Pull request by Zidious**
There was a shell script I created in https://github.com/dequelabs/axe-api-team-public/pull/104 that grabbed the new env vars for a given release that required a bit more testing. Instead of writing tests in shell, decided to move it over to TS.

Example run: https://github.com/dequelabs/zidious-testing/actions/runs/7252107557/job/19755876099 

No QA Required

### CODE REVIEW

File: .github/actions/get-new-env-vars-v1/src/run.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    const {
+      stdout: envVars,
+      stderr: envVarsError,
+      exitCode: envVarsExitCode
+    } = await getExecOutput(
+      `git diff origin/${base}...origin/${head} -- ${envFilePath}`
+    )
+
+    if (envVarsExitCode) {
+      throw new Error(`Error getting env vars: \n${envVarsError}`)
```
Comment (stephenmathieson) [inline .github/actions/get-new-env-vars-v1/src/run.ts:23]: It may be useful to print `stderr` here 🤷

Comment (Zidious) [reply .github/actions/get-new-env-vars-v1/src/run.ts:23]: `envVarsError` is `strderr` from the `getExecOutput` command 👀

Comment (straker) [reply .github/actions/get-new-env-vars-v1/src/run.ts:23]: The entire run function is in a try/catch and the catch outputs the error message and sets the run to fail

Comment (stephenmathieson) [reply .github/actions/get-new-env-vars-v1/src/run.ts:23]: > envVarsError is strderr from the getExecOutput command 👀

Ah, you're right. When skimming this, I thought you were including the exit code (not stderr). 👍

File: .github/actions/get-new-env-vars-v1/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "homepage": "https://github.com/dequelabs/axe-api-team-public#readme",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "ncc build src/index.ts --license ../../../licenses.txt",
+    "test": "mocha src/*.test.ts",
+    "coverage": "nyc npm run test",
+    "fmt": "prettier --write \"src/**/*.{js,ts,tsx,json,md,css,html}\"",
+    "lint": "eslint \"src/**/*.{js,ts,tsx}\""
+  },
+  "nyc": {
```
Comment (michael-siek) [inline .github/actions/get-new-env-vars-v1/package.json:23]: Should probably move this stuff to a `.npmrc` file but not a huge deal

**Review by michael-siek — APPROVED** [review]

File: .github/actions/get-new-env-vars-v1/package.json (hunk 2 above)
Comment (Zidious) [reply .github/actions/get-new-env-vars-v1/package.json:23]: Ahh you mean `.nycrc` file? 

https://github.com/dequelabs/axe-api-team-public/issues/109

**Pull request merged**